lib/ash/code_interface.ex

```diff
@@ -696,10 +710,11 @@ defmodule Ash.CodeInterface do
                         opts
                         |> Keyword.delete(:bulk_options)
                         |> Keyword.merge(Keyword.get(opts, :bulk_options, []))
+                        |> Keyword.put(:context, context)
                         |> Enum.concat(changeset_opts)
 
                       Ash.bulk_create(
-                        Stream.map(inputs, &Map.merge(&1, params)),
+                        inputs,
```

Comment:
Is this intentional?

Reply:
Yes, params and inputs are the same if the condition on line 679 passes and this gave us an error in our test.

Reply:
Ah, right because of this:

```elixir
            params =
              if is_list(params) do
                to_merge =
                  unquote(args)
                  |> Enum.zip([unquote_splicing(arg_vars)])
                  |> Map.new()

                Enum.map(params, fn params ->
                  Map.merge(params, to_merge)
                end)
              else
                unquote(args)
                |> Enum.zip([unquote_splicing(arg_vars)])
                |> Enum.reduce(params, fn {key, value}, params ->
                  Map.put(params, key, value)
                end)
              end
```

That handles merging args w/ params

Reply:
Ok, this comment can be marked as resolved then.